app/controllers/katello/api/v2/generic_content_units_controller.rb

```diff
@@ -2,7 +2,8 @@ module Katello
   class Api::V2::GenericContentUnitsController < Api::V2::ApiController
     resource_description do
       name 'Content Units'
-      param :content_type, String, desc: N_("Possible values: #{Katello::RepositoryTypeManager.generic_content_types.join(", ")}"), required: true
+      param :content_type, lambda { |val| Katello::RepositoryTypeManager.generic_content_types.include?(val) },
```

Comment:
This isn't only a docs fix, it also adds validation. I like that, but the commit message doesn't really reflect that.

Reply:
Added some extra bits to the message.